middlewares/authorization.js

```diff
@@ -0,0 +1,31 @@
+const jwt = require('jsonwebtoken');
+
+const authorizationMdware = (req, res, next) => {
+  try {
+    let token = req.headers['x-access-token'] || req.headers.authorization;
+    if (token) {
+      token = token.slice(7, token.length);
+      // Verify the token and if valid call next middleware
+      jwt.verify(token, process.env.SECRET, (err, authData) => {
```

Comment:
Please add an else here to handle in the outward scope, if a token was not handled successfully.